orca-bakery/src/test/groovy/com/netflix/spinnaker/orca/bakery/pipeline/BakeStageSpec.groovy

```diff
@@ -93,7 +99,9 @@ class BakeStageSpec extends Specification {
     }
 
     def bakeStage = pipeline.stageById("1")
-    def parallelStages = new BakeStage().parallelStages(bakeStage)
+    def bakeStageInstance = new BakeStage()
```

Comment:
should be able to `def parallelStages = new BakeStage(regionCollector: new RegionCollector().parallelStages(bakeStage)` if you wanted to preserve the existing form.

Reply:
nice, thanks!